cwd/app.js

```diff
@@ -123,16 +126,67 @@
     }
   };
 
+  /**
```

Comment:
Given that this code is presented as a diff here, I'd like to make the requirement that #19 gets merged before this can (otherwise we merge #19 by merging this).  Did you have any luck getting the testing / code review of #19 closed up?

Reply:
#19 has been reviewed.

Reply:
In theory this should now be resolved, but I decided that #19 has new requirements that should be met before it gets merged.  So, we can wait on the new migration system of sorts now. :/